lib/libxx/Kconfig

```diff
@@ -61,6 +61,11 @@ if LIBCXX
 config LIBCXX_EXCEPTION
 	bool "Enable Exception Suppport"
 	default y
+	--help--
+		Enable exception support in libc++.
+
+		NOTE: C++ exception support requires toolchain version 10.3.1. If
+		you are	enabling C++ exception then use docker image version 2.0.0.
```

Comment:
Hello @gSahitya-samsung , I have a minor question here. After this PR is merged, if there is some upgrades on other tools or package, then for GCC toolchain v6.3.1 based, the docker version will be eg. 1.5.9/1.6.0?
And for GCC toolchain v10.3.1, the docker version will be eg. 2.0.1/2.1.0?
As I understood, the change of major version is because of difference in GCC toolchain version. Then how will you address other changes in the future~? Will it be similar as mentioned above?

Reply:
Hello @edwakuwaku , yes, what you said is right. It's the reason why we left a lot of gap and went to 2.0.0 from 1.5.8 .

Fox example if some new package is added to docker, it will be added to both 6.3.1 and 10.3.1 docker images and we will update their version as you said to 2.0.1 and 1.5.9 respectively.

Reply:
Hiiiii @abhishek-samsung , thank you for confirming~